helm/mattermod/Chart.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: mattermod
+description: Auto-generates responses to GitHub issues and pull requests on mattermost/mattermost-server repository using the mattermod GitHub accounts
```

```suggestion
description: Auto-generates responses to GitHub issues and pull requests on mattermost/mattermost repository using the mattermod GitHub accounts
```

However, this is also used for other repos as well, right? Like mobile, desktop, etc.

---

Yes. 
This is actually a copy from this section : https://github.com/mattermost/mattermost-mattermod/blob/master/README.md?plain=1#L3 in Readme.md 

Last updated 5 years ago with : https://github.com/mattermost/mattermost-mattermod/commit/e7367dc38d0f8a5f613ff3c0d9c3bdb55e621bd8
 
Maybe this also needs update

---

@saturninoabril  Any objections in merging this as is, and updating the App description in a seperate PR?

---

Opss, sorry. Non-blocking and all good to me.